File: task2.js
const axios = require('axios');

axios.get('http://localhost:5000/api/data')
  .then(function (response) {
    var datalist = response.data.data
    datalist.sort(function (a, b) {
      return a.id - b.id;
    })
    for (var i = 0; i < datalist.length; i++) {
      console.log('item id : ' + datalist[i].id)
      console.log('item name : ' + datalist[i].name)
    }
  })
  .catch(function (error) {
    console.log(error);
  });

File: task2.py
import requests
url = 'http://localhost:5000/api/data'
res = requests.get(url)
print(res.json()['data'])

datalist = res.json()['data']
newlist = sorted(datalist, key=lambda d: d['id']) 
for item in newlist:
    print('item id : ', item['id'])
    print('item name : ',item['name'])
